frontend/src/components/PrevNext.js

```diff
@@ -1,16 +1,52 @@
 import React from 'react';
 import Data from './data.json'
-
+import './styles/BodyStyle.css'
+
+function openCloseAnswer(e)
+{
+  let element = e.target;
+  console.log(element)
+  if(element.tagName.toLowerCase() !== "div")
+    element = element.parentElement
+  element = element.nextElementSibling
+  
+  
+  const id = element.getAttribute("id").replace("A", "")
```

Comment:
You're removing `A` here.... why ? 🤔 
Is it related to `A`/`Q` if yes, please add a small comment explaining that... and why you're doing it